src/transport/mem.rs

```diff
@@ -300,6 +302,10 @@ impl<In: RpcMessage, Out: RpcMessage> crate::ServerChannel<In, Out, ChannelTypes
     fn accept_bi(&self) -> AcceptBiFuture<'_, In, Out> {
         AcceptBiFuture(self.stream.recv_async())
     }
+
+    fn local_addr(&self) -> &[crate::LocalAddr] {
+        &self.local_addr
```

Comment:
This will always be just LocalAddr::Mem, right?

Reply:
Yes, it is created on line 321.  This is a consequence of the `&[LocalAddr]` return type, I think this would be a more obvious if there was a `LocalAddrIter` or something that could be created with just the mem hardcoded here and without having to extend the struct itself.